alibuild_helpers/build.py

```diff
@@ -20,6 +20,7 @@
 from alibuild_helpers.log import logger_handler, LogFormatter, ProgressPrint, riemannStream
 from datetime import datetime
 from glob import glob
+from multiprocessing.pool import ThreadPool
```

Comment:
Done. But not sure, if it is doing, what you want, see https://pypi.python.org/pypi/multiprocessing/